benchmarks/src/main/scala/bloop/engine/tasks/TaskBenchmark.scala

```diff
@@ -0,0 +1,64 @@
+package bloop.engine.tasks
+
+import org.openjdk.jmh.annotations.Benchmark
+
+import scala.concurrent.Await
+import scala.concurrent.duration.Duration
+import bloop.engine.ExecutionContext.threadPool
+
+object TaskBenchmark {
```

Comment:
Are you gonna try with Monix?

Reply:
Maybe some time later, but I thought it didn't cost much to add a benchmark for the current tasks.